whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -6816,10 +6816,37 @@
     },
 
     "256": {
+      "NOTE:LC": "Prior to the definition of this field in 1987, file characteristics were recorded in subfield $a of field 300",
       "aboutEntity": "?thing",
-      "$a": {"link": "digitalCharacteristic", "resourceType": "FileType", "property": "label"}
+      "addLink": "digitalCharacteristic",
+      "resourceType": "DigitalCharacteristic",
+      "repeatable": false,
+      "$a": {"addLink": "hasNote", "resourceType": "Note", "property": "label", "NOTE:repeatable": false},
```

Comment:
I'd really prefer for this to either A) be reduced to just a `comment` property, or B) for this "sketchy" `digitalCharacteristic` to be changed to just a `hasNote` with type `marc:FileTypeNote`.

(Although the majority of data values seem to be a textual label for the `FileType` (somewhat in favour of the original mapping) I presume bib 256 is considered as legacy data which we might even drop the majority of?)

Reply:
It has now been updated per discussion.